app/server/api/blockchain_transaction_api.py

```diff
@@ -85,76 +85,17 @@ def post(self):
 
         call = post_data.get('call')
```

Comment:
this entire file is due for a review, because we don't use most of it anymore. Will do when i look into making sure new funds added to orgs are detected, since that's the functionality that overlaps with the old code here.